src/lib/proxy-server.ts

```diff
@@ -170,7 +170,8 @@ export const startProxyServer = sequential( async (): Promise< boolean > => {
 			const defaultOptions: https.ServerOptions = {
 				SNICallback: async ( servername, cb ) => {
 					try {
-						const site = await getSiteByHost( servername );
+						const punycodeServerName = domainToASCII( servername );
+						const site = await getSiteByHost( punycodeServerName );
```

Comment:
```suggestion
						const site = await getSiteByHost( servername );
```
`getSiteByHost` already converts the domain name to punycode internally.

Reply:
Great catch! I removed it.